test/e2e/node/crictl.go

```diff
@@ -33,42 +33,28 @@ var _ = SIGDescribe("crictl", func() {
 	ginkgo.BeforeEach(func() {
 		// `crictl` is not available on all cloud providers.
 		e2eskipper.SkipUnlessProviderIs("gce", "gke")
-		// The test requires $HOME/.ssh/id_rsa key to be present.
-		e2eskipper.SkipUnlessSSHKeyPresent()
 	})
 
 	ginkgo.It("should be able to run crictl on the node", func() {
-		// Get all nodes' external IPs.
-		ginkgo.By("Getting all nodes' SSH-able IP addresses")
-		hosts, err := e2essh.NodeSSHHosts(f.ClientSet)
-		if err != nil {
-			framework.Failf("Error getting node hostnames: %v", err)
-		}
-		e2eskipper.SkipUnlessAtLeast(len(hosts), 1, "Skipping test because found no SSH'able node")
+		nodes, err := e2enode.GetBoundedReadySchedulableNodes(f.ClientSet, maxNodes)
+		framework.ExpectNoError(err)
 
-		testCases := []struct {
-			cmd string
-		}{
-			{`sudo crictl version`},
-			{`sudo crictl info`},
+		testCases := []string{
+			"sudo crictl version",
+			"sudo crictl info",
```

Comment:
no need to sudo since the hostExec pod runs privileged and some OS doesn't have sudo (i.e. kind jobs)

https://github.com/kubernetes/kubernetes/blob/c82626f96f754e0eb87eaa1380bb672be42e6f0d/test/e2e/storage/utils/host_exec.go#L102-L106